cli/service.go

```diff
@@ -138,7 +138,7 @@ func getHandlerUI(settings *evergreen.Settings) (http.Handler, error) {
 	functionOptions := service.FuncOptions{
 		WebHome:  webHome,
 		HelpHome: settings.Ui.HelpUrl,
-		IsProd:   true,
+		IsProd:   settings.IsProd,
```

Comment:
make sure this doesn't default to false, might be good to invert this to be the other way around, so that it defaults to true, unless specifically set.

Reply:
Done